Automattic/simplenote-macos
# Updates Checklists Assets

### Details:
In this PR we're wiring the new Checklist Assets

@frosty May I bug you with a REALLY quick one?

Thanks James!!

### Test
1. Launch Simplenote
2. Add a new note
3. Insert a Checklist (**Format** > **Insert Checklist**)

- [ ] Verify that the checklist looks great
- [ ] Verify that clicking over the asset results in a checked state

### Release
`RELEASE-NOTES.txt` was updated in ff9e894 with:
 
> Updated Checklist Icons
